lib-exts/stdlib-meta/Reflection.agda

```diff
@@ -1,14 +1,27 @@
-{-# OPTIONS --safe #-}
-module Reflection.Ext where
+{-# OPTIONS --safe --without-K #-}
```

Comment:
Is it possible to use `--cubical-compatible` here?  Going forward I think should try to use `--cubical-compatible` instead of `--without-K` whenever possible.